recipes/root/all/conanfile.py

```diff
@@ -0,0 +1,305 @@
+
+    @property
+    def _configured_cmake(self):
+        if self._cmake is None:
+            self._move_findcmake_conan_to_root_dir()
+            self._cmake = CMake(self)
+            version = self.version.replace("v", "")
+            cmakelibpath = ";".join(self.deps_cpp_info.lib_paths)
+            cmakeincludepath = ";".join(self.deps_cpp_info.include_paths)
+            self._cmake.configure(
+                source_folder="root-{}".format(version),
+                defs={
+                    # TODO: Remove BUILD_SHARED_LIBS option when hooks issue is resolved
+                    # (see: https://github.com/conan-io/hooks/issues/252)
+                    "BUILD_SHARED_LIBS": "ON",
+                    "fail-on-missing": "ON",
+                    "CMAKE_CXX_STANDARD": self._cmake_cxx_standard,
+                    "gnuinstall": "OFF",
+                    "soversion": "ON",
+                    # Disable builtins and use Conan deps where available
+                    "builtin_pcre": "OFF",
+                    "builtin_lzma": "OFF",
+                    "builtin_zstd": "OFF",
+                    "builtin_lz4": "OFF",
+                    "builtin_glew": "OFF",
+                    "builtin_openssl": "OFF",
+                    "builtin_fftw3": "OFF",
+                    "builtin_cfitsio": "OFF",
+                    "builtin_davix": "OFF",
+                    "builtin_tbb": "OFF",
+                    # Enable builtins where there is no Conan package
+                    "builtin_xxhash": "ON",
+                    "builtin_afterimage": "ON",
+                    "builtin_gsl": "ON",
+                    "builtin_gl2ps": "ON",
+                    "builtin_ftgl": "ON",
+                    "builtin_vdt": "ON",
+                    # No Conan packages available for these dependencies yet
+                    "davix": "OFF",
+                    "pythia6": "OFF",
+                    "pythia8": "OFF",
+                    "mysql": "OFF",
+                    "oracle": "OFF",
+                    "pgsql": "OFF",
+                    "gfal": "OFF",
+                    "tmva-pymva": "OFF",
+                    "xrootd": "OFF",
+                    "pyroot": self._pyrootopt,
+                    # Tell CMake where to look for Conan provided depedencies
+                    "CMAKE_LIBRARY_PATH": cmakelibpath,
+                    "CMAKE_INCLUDE_PATH": cmakeincludepath,
+                    # Configure install directories
+                    # Conan CCI hooks restrict the allowed install directory
+                    # names but ROOT is very picky about where build/runtime
+                    # resources get installed.
+                    # Set install prefix to work around these limitations
+                    # Following: https://github.com/conan-io/conan/issues/3695
+                    "CMAKE_INSTALL_PREFIX": "{}{}res".format(
+                        self.package_folder, os.sep
+                    ),
```

Comment:
Usually I see `os.path.join` for then `format(os.sep)` 

#3614 anything item?

Reply:
c4a2736 now uses `os.path.join` here.

> #3614 anything item?

Sorry, I don't understand what you are asking here.

Reply:
Sorry I've been keeping a list of all the tiny reviewing comments that come up which go for the PR above